src/mats_l1_processing/L1_calibrate.py

```diff
@@ -105,8 +109,8 @@ def L1_calibrate(CCDitem, instrument, force_table: bool = True):  # This used to
 
     CCDitem["image_calibrated"] = image_calibrated
 
-    errors = combine_flags([error_bad_column,error_flags_bias,error_flags_linearity,error_flags_desmear,
+    errors = combine_flags([error_bad_column,error_flags_se, error_flags_hp, error_flags_bias,error_flags_linearity,error_flags_desmear,
     error_flags_dark,error_flags_flatfield,error_ghost,error_artifact],
-    [1,1,2,1,3,2,1,2])
+    [1,1, 1,1,2,1,3,2,1,2])
     
-    return image_lsb, image_bias_sub, image_desmeared, image_dark_sub, image_calib_nonflipped, image_calib_flipped, image_calibrated, errors
+    return image_lsb, image_se_corrected, image_hot_pixel_corrected, image_bias_sub, image_desmeared, image_dark_sub, image_calib_nonflipped, image_calib_flipped, image_calibrated, errors
```

Comment:
```suggestion
    return (
        image_lsb,
        image_se_corrected,
        image_hot_pixel_corrected,
        image_bias_sub,
        image_desmeared,
        image_dark_sub,
        image_calib_nonflipped,
        image_calib_flipped,
        image_calibrated,
        errors,
    )
```
This row is getting ridiculous! ;)

Not that added outputs will require changes in the Lambda as well. I will try to remember to add space for them or refactor to accommodate in a more sustainable way.

Reply:
Whatever you prefer, I can work with! I do think you should break that long line if you want to keep all outputs though!

Reply:
Well, we are not using them in the processing, but it is good that they are outputed when you want to run the calibration offline and check if, and in that case where, something has gone wrong.